conda/environments/triton_test_no_client.yml

```diff
@@ -0,0 +1,19 @@
+name: triton_test
+channels:
+  - conda-forge
+  - nvidia
+  - rapidsai
+dependencies:
+  - aws-sdk-cpp<1.9
+  - clang-tools=11.1.0
+  - cudatoolkit=11.4
+  - cuml=21.12
+  - flake8
+  - hypothesis
+  - lightgbm
+  - pip
+  - pytest
+  - python=3.8
+  - rapidsai::py-xgboost
+  - scikit-learn
+  - treelite
```

Comment:
Treelite is a cuml dependency, you could skip it here if needed

Reply:
I think I'd prefer to keep it, since it's a direct dependency of our tests. Can't imagine cuML would ever drop Treelite, but if it did, we'd still need it here.